_includes/markdown/CSS.md

```diff
@@ -4,377 +4,578 @@ At 10up, we value content and the experience users will have reading it. We writ
+Content is unpredictable. You might not know how long a title will be, how many items a navigation menu will have, or the length of an excerpt in a card. Design decisions made today can quickly be overturned tomorrow when new requirements come in.
 
-### Min-width media queries
+Prepare for edge cases. Your components must be resilient and robust enough to handle all reasonable variations, including unexpected ones. If you don’t account for these possibilities, it could lead to broken layouts, unreadable text, or poor user experiences.
 
-A responsive website should be built with min-width media queries. This approach means that our media queries are consistent, readable and minimize selector overrides.
+### Stress Test Your Components
 
-* For most selectors, properties will be added at later breakpoints. This way we can reduce the usage of overrides and resets.
-* It targets the least capable browsers first which is philosophically in line with mobile first — a concept we often embrace for our sites
-* When media queries consistently "point" in the same direction, it makes it easier to understand and maintain stylesheets.
+When building a component, test it against extreme content scenarios. For example:
 
-Avoid mixing min-width and max-width media queries.
+* A navigation menu with 20 items instead of 5\.  
+* A card title that wraps onto three or more lines.  
+* A card title that is just one word  
+* An excerpt that’s 300 characters long—or empty.
 
-### Breakpoints
+Building flexible, content-agnostic components makes your work more resilient to change. It reduces the need for constant adjustments and protects the layout from breaking when new content doesn’t fit the original plan. And let’s be real: the original plan often changes.
 
-Working with build tools that utilize Sass or PostCSS processing, we can take advantages of reusability and avoid having an unmaintainable number of breakpoints. Using variables and reusable code blocks we can lighten the CSS load and ease maintainability.
+<h2 id="minimal-styles" class="anchor-heading">Write Only the Styles You Need, Don't Do Too Much {% include Util/link_anchor anchor="minimal-styles" %}</h2>
 
-### Media queries placement
+When writing CSS, stick to the styles you need and avoid overcomplicating. For example, centering a container horizontally often involves a common snippet:
 
-In your stylesheet files, nest the media query within the component it modifies. **Do not** create size-based partials (e.g. `_1024px.(s)css`, `_480px.(s)css`): it will be frustrating to hunt for a specific selector through all the files when we have to maintain the project. Putting the media query inside the component will allow developers to immediately see all the different styles applied to an element.
+```css
+.container {
+  max-inline-size: 48rem;
```

Comment:
This is something I don't agree with too much.  Mostly because this is transpiled by PostCSS Preset Env most of the time. This introduces cognitive load into the developer for a use case that I don't think we have ever seen (in which `inline-size` means something different than `width`)